airbyte-ci/connectors/metadata_service/lib/metadata_service/commands.py

```diff
@@ -34,10 +34,11 @@ def validate(file_path: pathlib.Path):
 @metadata_service.command(help="Upload a metadata YAML file to a GCS bucket.")
 @click.argument("metadata-file-path", type=click.Path(exists=True, path_type=pathlib.Path))
 @click.argument("bucket-name", type=click.STRING)
-def upload(metadata_file_path: pathlib.Path, bucket_name: str):
+@click.option("--prerelease", type=click.STRING, required=False, default=None, help="The prerelease tag of the connector.")
+def upload(metadata_file_path: pathlib.Path, bucket_name: str, prerelease: str):
```

Comment:
oh! now we get to pick our pre-release tag, or is this just `--prerelease` signaling that this is a prerelease action?  I think I'm reading this as 
* `--prerelease` (no value) means this is a pre-release and to make a default tag from the git sha
* `--prerelease foo` means to make a docker image tagged `dev-foo` (the dev is always there)

Reply:
Ella you got it!

This is just for the metadata_service cli (fun fact that this is a cli tool installed and use by the `airbyte-ci` tool)

So for the metadata service you can upload a given metadata file under a prerelease location